package.json

```diff
@@ -94,6 +95,7 @@
     "titleTemplate": "{title}"
   },
   "dependencies": {
+    "ethers": "^5.1.4",
     "js-sha3": "^0.8.0",
     "typescript": "^4.1.5",
     "web3": "^1.3.5 ",
```

Comment:
I thought we were removing `web3` entirely. Can we drop this dependency and the one after it?

Reply:
good point.